src/main/java/pl/polsl/egradebook/model/entities/Case.java

```diff
@@ -0,0 +1,78 @@
+package pl.polsl.egradebook.model.entities;
+
+import javax.persistence.*;
```

Comment:
I think its to use: 
import javax.persistence.Entity;
import javax.persistence.Id;
import javax.persistence.Table;